docs/tutorial/index.rst

```diff
@@ -1,4 +1,147 @@
 Tutorial
 ========
 
-TBD
+Welcome! You are the head scheduler for the upcoming :code:`name` con. You have
+a venue, you have talks and you have a week to schedule everything!
+
+Your venue has two rooms in which you will schedule talks and workshops in
+parallel but you're also going to want to find time for some social events.
+
+You have organised your time slots as follows:
+
+- The first day will have 2 sessions (morning and afternoon) with 2 30 minute
+  time slots in each room.
+- The second day will have 1 room used for longer 1 hour workshops, the other
+  room used for more talks and 2 long sessions set aside for the social events.
+
+Let us create these time slots using the :code:`conference_scheduler`::
+
+    >>> from conference_scheduler.resources import Room, Slot, Event
+
+    >>> rooms = [Room(name="Big", capacity="200", suitability=[]),
+    ...          Room(name="Small", capacity="50", suitability=[]),
+    ...          Room(name="outside", capacity="1000", suitability=[])]
```

Comment:
The capacity being used in the rooms is not actually being used anywhere. This is just a place holder for now: I think it opens up a discussion about whether or not we need it but that's for another issue.